Makefile

```diff
@@ -227,7 +227,8 @@ pause: pause/pause.go
 	pause \
 	show-header \
 	show-summary \
-	show-variables
+	show-variables \
+	install-git-hooks
```

Comment:
This list looks to be alphabetical - can you move your addition up to maintain that?